### PR TITLE
feat: add inspector builder doc

### PR DESCRIPTION
**Pull request by sanyuan0704**
# PR Details

## Types of changes

- [ ] Docs change / refactoring / dependency upgrade
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist

- [ ] My code follows the code style of this project.
- [ ] I have updated changeset
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: packages/builder/builder-doc/zh/config/performance/bundleAnalyze.md [hunk 1]
```diff
@@ -1 +1,28 @@
-TODO
+你有两种方式开启 `webpack-bundle-analyzer` 来分析打包结果:
```
Comment (chenjiahan) [inline packages/builder/builder-doc/zh/config/performance/bundleAnalyze.md:1]: Type 和 Default 也写一下~

File: packages/builder/builder-doc/zh/config/output/polyfill.md [hunk 2]
```diff
@@ -1 +1,26 @@
...
+
+通过 `output.polyfill` 你可以配置 Polyfill 的注入方式。
+
+### 配置项
+
+#### entry
+
+当 `output.polyfill` 配置为 `'entry'` 时，会在每个入口文件中注入 Polyfill。
+
+等于与 `@babel/preset-env` 的 `useBuiltIns: 'entry'` 配置。
```
Comment (chenjiahan) [inline packages/builder/builder-doc/zh/config/output/polyfill.md:12]: 等于与 -> 等价与

File: packages/builder/builder-doc/zh/config/tools/inspector.md [hunk 3]
```diff
@@ -1 +1,38 @@
-TODO
+- 类型: `Object` | `Function` | `undefined`
+- 默认值: `undefined`
```
Comment (chenjiahan) [inline packages/builder/builder-doc/zh/config/tools/inspector.md:2]: 类型/默认值 统一用 Type/Default 表述哈

File: packages/builder/builder-doc/zh/config/tools/inspector.md [hunk 4]
```diff
@@ -1 +1,38 @@
...
+
+你可以通过 `tools.inspector` 开启或者配置 Webpack 调试工具 [webpack-inspector](https://github.com/modern-js-dev/webpack-inspector)。
+
+当配置不为 `undefined` 时，则表示开启了 `webpack-inspector`，此时 `tools.inspector` 的类型可以为 `Object` 或者 `Function`。
+
+## 类型
```
Comment (chenjiahan) [inline packages/builder/builder-doc/zh/config/tools/inspector.md:8]: h2 -> h3

File: packages/builder/builder-doc/zh/config/performance/bundleAnalyze.md (hunk 1 above)
Comment (sanyuan0704) [reply packages/builder/builder-doc/zh/config/performance/bundleAnalyze.md:1]: 这个不太好描述，可以看看上下文

Comment (sanyuan0704) [reply packages/builder/builder-doc/zh/config/performance/bundleAnalyze.md:1]: 后面写了不同情况的默认值

Comment (chenjiahan) [reply packages/builder/builder-doc/zh/config/performance/bundleAnalyze.md:1]: 了解，那写个 Type 就好

**Review by chenjiahan — APPROVED** [review]

**Pull request merged**